Core/src/org/sleuthkit/autopsy/datamodel/tags/Category.java

```diff
@@ -16,7 +16,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.sleuthkit.autopsy.imagegallery.datamodel;
+package org.sleuthkit.autopsy.datamodel.tags;
```

Comment:
I think this class is more general purpose. Rather than put it in a new package, I think it can go into the datamodel package, just with a more specific name: DhsImageCategory.